mungegithub/mungers/approvers/owners.go

```diff
@@ -452,15 +453,15 @@ func (ap Approvers) GetCCs() []string {
 	return suggested.Union(keepAssignees).List()
 }
 
-// IsApproved returns a bool indicating whether or not the PR is approved
-func (ap Approvers) IsApproved() bool {
+// AreFilesApproved returns a bool indicating whether or not the PR is approved
+func (ap Approvers) AreFilesApproved() bool {
 	return ap.UnapprovedFiles().Len() == 0
 }
 
-// IsApprovedWithIssue verifies that the PR is approved, and has an
-// associated issue or a valid "no-issue" approver.
-func (ap Approvers) IsApprovedWithIssue() bool {
-	return ap.IsApproved() && (ap.AssociatedIssue != 0 || len(ap.NoIssueApprovers()) != 0)
+// IsApproved verifies that the PR is approved, and if an associated issue is required
+// - that there is an associated issue or a valid "no-issue" approver.
```

Comment:
What does `-` mean here? Could you rephrase that comment?

Even the first part of the sentence is not really correct now:
> IsApproved verifies that all files in the PR have been approved, and ...